examples/kmm-sample/androidApp/src/main/AndroidManifest.xml

```diff
@@ -22,10 +22,9 @@
             android:allowBackup="false"
             android:supportsRtl="true"
             android:theme="@style/AppTheme">
-        <activity android:name="io.realm.example.kmmsample.androidApp.MainActivity">
+        <activity android:name="io.realm.example.kmmsample.androidApp.MainActivity" android:exported="true">
```

Comment:
We don't have any broadcast receiver, why add `exported=true`?

Reply:
It is now required by Android 12 for all activities started from the launcher.